var/spack/repos/builtin/packages/wget/package.py

```diff
@@ -39,6 +39,7 @@ class Wget(Package):
     version('1.16', '293a37977c41b5522f781d3a3a078426')
 
     depends_on("openssl")
+    depends_on("perl@5.12.0:", type=('build'))
```

Comment:
You're right, I'll change it. It's a habit I'm in to remind myself that it should be a tuple.